bot/config.py.zephyr.sample

```diff
@@ -85,7 +85,15 @@ z['gdrive'] = {
 
 z['iut_config'] = {
     "prj.conf": {},  # Default config file name
-
+    "eatt.conf": {
+        "overlay": {
```

Comment:
So we need eatt for all GATT test cases?

Reply:
Yes, because if it's enabled in ICS, it's chosen as bearer for all test cases, as per GATT TS, 4.2.1 ATT and EATT Bearers in Setup Preambles. Maybe we should just make PR to Zephyr with changed basic config?

Reply:
And in this PR just leave added wid 402 and edited IXIT